pkg/constants/constants.go

```diff
@@ -95,5 +95,9 @@ const (
 var (
 	ComponentDefaultLabel         = map[string]string{"e2e-test": "true"}
 	ComponentDefaultAnnotation    = map[string]string{ComponentInitialBuildAnnotationKey: "processed"}
-	ComponentPaCRequestAnnotation = map[string]string{"appstudio.openshift.io/pac-provision": "request"}
+	ComponentPaCRequestAnnotation = map[string]string{
+		"appstudio.openshift.io/pac-provision": "request",
+		"image.redhat.com/generate":            "true",
+		"image.redhat.com/delete-image-repo":   "true",
```

Comment:
please create a different variable for image-controller-specific annotations

Reply:
or similar to how `ComponentDefaultAnnotation` is defined ..  using`ComponentInitialBuildAnnotationKey` variable for the annotation?

Reply:
I meant adding a new variable for image-controller annotations
```go
ComponentWithImageControllerAnnotation = map[string]string{
		"image.redhat.com/generate":            "true",
		"image.redhat.com/delete-image-repo":   "true",
}
```
and then merging it with `ComponentPaCRequestAnnotation ` during the component creation